src/main/java/uk/gov/hmcts/darts/transcriptions/enums/TranscriptionTypeEnum.java

```diff
@@ -18,4 +21,13 @@ public enum TranscriptionTypeEnum {
     OTHER(999);
 
     private final Integer id;
+
+    public static TranscriptionTypeEnum fromId(Integer id) {
+        for (TranscriptionTypeEnum b : TranscriptionTypeEnum.values()) {
+            if (b.id.equals(id)) {
+                return b;
+            }
+        }
+        throw new DartsApiException(BAD_REQUEST_TRANSCRIPTION_TYPE);
```

Comment:
It was originally in the code before I changed it. It will be needed for the next step in the service which will call it in the same way as the tests.

Reply:
Don't understand this comment @karen-hedges 
https://github.com/search?q=repo%3Ahmcts%2Fdarts-api%20BAD_REQUEST_TRANSCRIPTION_TYPE&type=code

Reply:
The TranscriptionTypeEnum will be passed into the validator. This means the transcription that returns the entity needs to be converted to the enum. This method will provide that means.

Reply:
I guessed that - yep :)